### PR TITLE
added lgamma to tensorflow frontend, backend implementation missing

### LINKED ISSUE #18234: lgamma

### PR DESCRIPTION
**Pull request by marccgrau**
Close #18234

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 1]
```diff
@@ -465,6 +465,35 @@ def test_tensorflow_negative(
     )
 
 
+# lgamma
+@handle_frontend_test(
+    fn_tree="tensorflow.math.lgamma",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:472]:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
`valid` should be able to return all the supported dtypes for the function.

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Thanks for the PR! Just requested some changes. Please feel free to request another review when you are done :)

**Comment by hello-fri-end** [comment]
Hey @marccgrau! Looks like we don't have an `ivy.lgamma` function yet. In this case, you can implement it as a composition of other functions in the frontend unless the implementation takes more than a few lines in which case you should first implement the `lgamma` function in the functional API. Take a look at https://unify.ai/docs/ivy/overview/contributing/open_tasks.html for more details. Thanks :)

**Comment by marccgrau** [comment]
Hi @hello-fri-end 
Thanks for the review! I think a compositional implementation is not feasible as the gamma function is missing in ivy.
I opened a new issue (#18985) to add `ivy.lgamma` to the functional API. Several implementations in the native backends are available.

pending #18985

**Comment by hello-fri-end** [comment]
> Hi @hello-fri-end Thanks for the review! I think a compositional implementation is not feasible as the gamma function is missing in ivy. I opened a new issue (#18985) to add `ivy.lgamma` to the functional API. Several implementations in the native backends are available.
> 
> pending #18985

Sounds good @marccgrau ! Thanks for taking the initiative of adding `lgamma` to the functional API :rocket:

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by marccgrau** [comment]
Hi @hello-fri-end, `ivy.lgamma` is now included in the backend (#19152). Thus, I assume the frontend function could now be included.

**Comment by hello-fri-end** [comment]
> Hi @hello-fri-end, `ivy.lgamma` is now included in the backend (#19152). Thus, I assume the frontend function could now be included.

Awesome! Again, thanks for taking the initiative of adding it in ivy :rocket: 

Would you please sync this PR with master so that the changes are reflected here as well?

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**